content/auth/token.textile

```diff
@@ -480,6 +557,49 @@ $signature = hash_hmac(
 $jwt = $base64Header . '.' . $base64Claims . '.' . $signature;
 ```
 
+```[go]
```

Comment:
This is a code representing server side construction of `jwt` token. 
I would also recommend adding client side code that uses `jwt` token using `authCallback`.
You can refer to https://github.com/ably/ably-dotnet?tab=readme-ov-file#using-the-token-auth-at-client-side

Reply:
Using `jwt` at client side is tricky and depends on given SDK. 
That's why I strongly feel we include client code for each SDK that uses JWT string token using `authCallback`

Reply:
I don't think that is an update for this pull request, but rather similar to the PHP PR, an addition that can improve the authentication section. Something we have in the pipeline. I've added this to the Authentication documentation update ticket: https://ably.atlassian.net/browse/EDU-1555

Reply:
Okay, I can see it as a comment . 
Please do make sure this is addressed in the future for all SDKs 👍